scripts/variable.sh

```
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#First line is Shebang line which will indicate this script will use bash shell to execute shell commands
+
+
+<<multi_line_comments
```

Shellcheck

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)